src/config.ts

```diff
@@ -9,15 +11,27 @@ export interface ImportSpec {
   readonly source: string;
 }
 
+export interface ValidationConfig {
+
+  readonly package: string;
+  readonly version: string;
+  readonly class: string;
+  readonly installEnv?: { [key: string]: any };
+  readonly properties?: { [key: string]: any };
+}
+
 export interface Config {
   readonly app?: string;
   readonly language?: Language;
   readonly output?: string;
   readonly imports?: string[];
+  readonly pluginsDirectory?: string;
+  readonly validations?: string | ValidationConfig[];
 }
 
 const DEFAULTS: Config = {
   output: 'dist',
+  pluginsDirectory: path.join(os.homedir(), '.cdk8s', 'plugins'),
```

Comment:
That's very Linux-y, but going to be awkward on Windows, and (but somewhat less) macOS. I'd recommend platform-specific defaults (although feel free to ignore me here)

Reply:
(going to ignore :))